.github/workflows/bumpdevversion.yml

```
@@ -0,0 +1,124 @@
+on:
+  push: []
+    # branches:
+    #   - devel
+  # Schedule runs on 9am every Tuesday
+  schedule:
+    - cron: '0 9 * * 2'
+
+jobs:
+  bump-dev-version:
+    name: Bump GMT development version
+    runs-on: ubuntu-latest
+    if: github.repository == 'weiji14/gmt-feedstock'
+
+    steps:
+      # Generate token from GenericMappingTools bot
+      # - uses: tibdex/github-app-token@v1.3.0
+      #   id: generate-token
+      #   with:
+      #     app_id: ${{ secrets.APP_ID }}
+      #     private_key: ${{ secrets.APP_PRIVATE_KEY }}
+
+      # Checkout current git repository
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+        with:
+          # token: ${{ steps.generate-token.outputs.token }}
+          repository: conda-forge/gmt-feedstock
+          ref: devel
+
+      # Install yq - A portable command-line YAML processor
+      - name: Install yq
+        run: |
+          sudo wget -O /usr/local/bin/yq https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64
```

```suggestion
          sudo wget -O /usr/local/bin/yq https://github.com/mikefarah/yq/releases/download/v4.6.1/yq_linux_amd64
```
There's a new v4 version of `yq` which has some significant syntax changes from v3 (see https://mikefarah.gitbook.io/yq/upgrading-from-v3). Might need to spend some time updating the script.